adversarial-attacks/evasion-attack.ipynb

```diff
@@ -1707,6 +1748,22 @@
    "source": [
     "As we can see, the performance metrics are nearly restored to their values before the evasion attack ([see original metrics of the model](#metrics)). We may conclude that the implemented defence, i.e. adversarial training, increased the robustness of the model against adversarial attacks."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Self-check questions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. What methods of defence against adversarial attacks do you know?\n",
+    "2. We have implemented an adversarial attack and protected the model from it. What happens if the protected model is attacked by the second iteration of the same attack?\n",
+    "3. Try to implement 5 iterations of attack and defense with tracking of model quality metrics: before the attack, after the attack, after the defenсe. Give recommendations to the model developer on how to ensure the stability of the model under possible iterative attacks."
```

Comment:
I'm not sure "stability" is a generally accepted term. I suggest replacing: "to ensure the stability of the model under possible iterative attacks" with "to ensure the robustness of the model against possible iterative attacks".